packages/zengl/test_zengl.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from pytest_pyodide import run_in_pyodide
+
+
+@pytest.mark.xfail_browsers(node="this supposed to render into a canvas DOM element")
+@run_in_pyodide(packages=["zengl"])
+def test_render_with_webgl2(selenium_standalone):
```

Comment:
looking at some others tests..., should this be just `selenium`?

Reply:
Yeah seems like most use `selenium` and that will be faster so probably we should stick with `selenium` unless there is a specific problem with it.